components/search/Search.vue

```diff
@@ -176,6 +183,13 @@ export default class Search extends mixins(
     this.updateSearch(value)
   }
 
+  @Watch('name', { deep: true })
+  protected onSearchQueryChange(value) {
+    if (value) {
+      this.updateSearch(value)
+    }
+  }
```

Comment:
Which problem do these codes solve? I guess it's not that necessary.

Reply:
so currently, if you just write something, the url doesn't get updated and then you click on see all, nothing happens
this will update the url, so that when you click on see all, you see the correct results

Reply:
However, these changes would bring an extra problem. Whenever I type a single character, the search page would flush. We prefer the search page update when there is a `enter` or `blur` event. 
![Kapture 2023-02-26 at 10 49 47](https://user-images.githubusercontent.com/31397967/221389544-7b3c9cbc-6ddb-4752-901d-8e43c1309aa6.gif)

So I guess the better approach is to update the url query when you click see all. ( check `SearchSuggestion.vue`)

Reply:
okay yeah makes sense, I'll update on see all then

Reply:
updated